OWML.Common/IModConsole.cs

```diff
@@ -1,8 +1,12 @@
-﻿namespace OWML.Common
+﻿using System;
+
+namespace OWML.Common
 {
     public interface IModConsole
     {
-        void WriteLine(string s);
```

Comment:
this breaks backwards compatibility?

Reply:
shouldnt do, as the new method is `void WriteLine(string line, MessageType type = MessageType.Message);`
(Admittedly I didn't have the `= MessageType.Message` in there when you reviewed, so that's on me :P)

Reply:
i've tried adding default parameters to existong methods before, it breaks backwards compatibility, even if it's not obvious

Reply:
well all the OWML log messages, and the ones from the example mods work

Reply:
yeah but with old mods?

Reply:
that's because they got recompiled

Reply:
fixed 👍